internal/person.go

```diff
@@ -0,0 +1,21 @@
+package internal
+
+type Person struct {
+	CompareValue   string
+	ID             string
+	Attributes     map[string]string
+	DisableChanges bool
+}
+
+func (p *Person) Matches(filters Filters) bool {
+	for _, f := range filters {
+		value := p.Attributes[f.Attribute]
```

Comment:
Do we want to make this case insensitive? I'm fine either way.

Reply:
Not necessarily, f.Attribute could be `Foo` but p.Attributes could contain `foo`, in which case, I don't think it would find it.

Reply:
> Not necessarily, f.Attribute could be `Foo` but p.Attributes could contain `foo`, in which case, I don't think it would find it.

Oh, you're right. I wasn't thinking of the attribute name. But I think case-sensitive is better. It's pretty rare that case is ignored in json.